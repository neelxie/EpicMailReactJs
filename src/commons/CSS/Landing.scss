.brand {
    margin: auto;
    width: 80%;
    font-family: roboto;
    font-size: 30px;

    p {
        color: wheat;
    }
}