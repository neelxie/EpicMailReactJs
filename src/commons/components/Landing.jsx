/* eslint-disable import/no-extraneous-dependencies */
import React from 'react';
import HeaderSection from './HeaderSection';
import '../CSS/Landing.scss';
export default function Landing() {
  return (

    <div className="mainPage">

      <HeaderSection />
      <div className="brand">
        <p>The internet is increasingly becoming an integral part of lives.
        Ever since the invention of electronic mail by Ray Tomlinson,
        emails have grown to become the primary medium of exchanging
        information over the internet between two or more people,
        until the advent of Instant Messaging (IM) Apps.</p>

        <p>As EPIC Andelans who work towards advancing human potential 
        and giving back to the society, we wish to empower others by
        having a web app that helps people exchange 
        messages/information over the internet.</p>
        
      </div>
    </div>
  );
}
