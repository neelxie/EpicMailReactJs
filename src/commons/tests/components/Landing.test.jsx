/* eslint-disable no-undef */
import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Landing from '../../components/Landing';

// test
describe('<Landing> Component', () => {
  it('should match the snapshot', () => {
    const component = shallow(<Landing />);

    expect(toJson(component)).toMatchSnapshot();
  });
});
