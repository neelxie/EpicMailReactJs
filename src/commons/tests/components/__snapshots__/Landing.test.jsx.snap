// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Landing> Component should match the snapshot 1`] = `
<div
  className="mainPage"
>
  <HeaderSection />
  <div
    className="brand"
  >
    <p>
      The internet is increasingly becoming an integral part of lives. Ever since the invention of electronic mail by Ray Tomlinson, emails have grown to become the primary medium of exchanging information over the internet between two or more people, until the advent of Instant Messaging (IM) Apps.
    </p>
    <p>
      As EPIC Andelans who work towards advancing human potential and giving back to the society, we wish to empower others by having a web app that helps people exchange messages/information over the internet.
    </p>
    <p>
      So join the revolution signup or login now.
    </p>
  </div>
</div>
`;
